stdlib/runtime/compare.gr

```diff
@@ -132,6 +133,18 @@ let rec heapCompareHelp = (heapTag, xptr, yptr) => {
 
       return 0
     },
+    t when t == Tags._GRAIN_INT32_HEAP_TAG => {
+      let xval = WasmI32.load(xptr, 4n)
+      let yval = WasmI32.load(yptr, 4n)
+      return if (WasmI32.ltS(xval, yval)) -1 else if (WasmI32.gtS(xval, yval)) 1
+      else 0
+    },
+    t when t == Tags._GRAIN_FLOAT32_HEAP_TAG => {
+      let xval = WasmF32.load(xptr, 4n)
+      let yval = WasmF32.load(yptr, 4n)
+      return if (WasmF32.lt(xval, yval)) -1 else if (WasmF32.gt(xval, yval)) 1
+      else 0
+    },
```

Comment:
This needs to be updated to handle NaN properly, with NaN being equal to itself (should already be correct here) and smaller than any other number (not handled here). Would be good to add a few tests as well.